RELEASES.md

```diff
@@ -16,6 +16,7 @@ Changed default log level from DEBUG to INFO, for environments that are not conf
 - Transaction Manager BTM is removed. Switch over to Narayana Transaction Manager.
 - Only supports Tomcat 10.x or later. Tomcat 9.x or lower version, are no longer supported.
 - By default the PipelineSession substitution delimiter has been changed from `${` to `?{` so it's consistent with the `FixedQuerySender`. Backwards compatibility key `useOldSubstitutionStartDelimiter` has been added so minimal change is required during upgrades. Note that when using caches in combination with `diskPersistent="true"` you may need to purge your cache!
+- FileSystemPipes and FileSystemSenders now have new forwards for `fileNotFound`, `folderNotFound`, `fileAlreadyExists`, `folderAlreadyExists`. Some actions, such as removing a non-existing folder, were previously ignored but can now trigger one of these forwards. If such a forward is not defined, then the pipe or sender will go to the `exception` forward or if that is not defined either, trigger an exception, which was previously ignored. These configurations need to be checked.
```

Comment:
Good explanation. If I was an Integration Noob, I'd also like to get a suggestion on how to reach the same state on ignoring these exceptions. So, should I add an empty exception forward? Example e.g.?
Or is `test/src/main/configurations/MainConfig/ConfigurationMoveFilePipe.xml` a good example?

Reply:
Hmm, adding the exception (or more specific 'fileAlreadyExists') forward with a path to the next expected forward should fix this. The path may never be empty.
Perhaps in stead of 'These configurations need to be checked.' you could write 'Adding the (specific) exception forward and point it to the next pipe will solve this.'

Reply:
Hm. Some of that should go into general documentation / teaching and not in the upgrade notes.
But a generic `exception` forward will also trigger for other problems that happen, so it can be too broad.